server/server.go

```diff
@@ -94,6 +94,7 @@ var (
 	errAccessDenied            = terror.ClassServer.New(errno.ErrAccessDenied, errno.MySQLErrName[errno.ErrAccessDenied])
 	errConCount                = terror.ClassServer.New(errno.ErrConCount, errno.MySQLErrName[errno.ErrConCount])
 	errSecureTransportRequired = terror.ClassServer.New(errno.ErrSecureTransportRequired, errno.MySQLErrName[errno.ErrSecureTransportRequired])
+	errMultiStatementDisabled  = terror.ClassServer.New(errno.ErrUnknown, "client has multi-statement capability disabled") // MySQL returns a parse error
```

Comment:
Is it easy for you to make it consistent with MySQL?

Reply:
MySQL only returns a generic parse error, and doesn't make it easy to understand why it did so (the multi-statement option was not set). In TiDB the statement is already parsed, so I did this because I think it's an improvement.

Reply:
OK.